test/src/integration/integration_test.go

```diff
@@ -154,25 +161,47 @@ func runAnsiblePlaybooks(t *testing.T) {
 
 	shell.RunCommand(t, replaceCommand)
 
-	err = ioutil.WriteFile("./kube_config", []byte(lookupTargetValue(t, "kubernetes", "kube_config")), 0644)
+	err = files.CopyFile("../../modules/"+cloudProvider+"/kubernetes/kube_config", "./kube_config")
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	err = ioutil.WriteFile("./inventory.ini", []byte(lookupTargetValue(t, "kubernetes", "inventory_ini")), 0644)
+	err = files.CopyFile("../../modules/"+cloudProvider+"/network/network_inventory", "./inventories/network_inventory")
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	// Install tools
-	runAnsiblePlaybook(t, k8sModuleDir, []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../", "-e", "install_awscli=" + installAwscli})
+	runAnsiblePlaybook(t, k8sModuleDir, "../inventories", []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../", "-e", "install_awscli=" + installAwscli})
```

Comment:
Maybe this shouldn't be done in thin PR but `install_awscli` option is too specific so it should be changed to something like `cloud=aws/azure` so that cloud-specific provisioning can be generally done in the ansible script.

Reply:
Fixed with https://github.com/scalar-labs/scalar-kubernetes/pull/112

Reply:
Nice! Thank you!